src/components/VideoLibraryModal.vue

```diff
@@ -73,71 +81,187 @@
+                    v-if="selectedVideos.find((v) => v.fileName === video.fileName) && !isMultipleSelectionMode"
                     class="play-button"
-                    @click="playVideo"
+                    @click="video.isProcessed ? playVideo() : processSingleVideo()"
                   >
-                    <v-icon size="40" class="text-white">mdi-play-circle-outline</v-icon>
+                    <v-icon size="40" class="text-white">{{
+                      video.isProcessed ? 'mdi-play-circle-outline' : 'mdi-progress-alert'
+                    }}</v-icon>
                   </div>
                   <div
-                    v-if="selectedVideos.find((v) => v.filename === video.filename) && isMultipleSelectionMode"
+                    v-if="isMultipleSelectionMode"
                     class="checkmark-button"
+                    :class="selectedVideos.find((v) => v.fileName === video.fileName) ? 'bg-green' : 'bg-white'"
+                    @click.stop="toggleVideoIntoSelectionArray(video)"
                   >
-                    <v-icon size="15" class="text-white">mdi-check-circle-outline</v-icon>
+                    <v-icon size="15" class="text-white">
+                      {{
+                        selectedVideos.find((v) => v.fileName === video.fileName)
+                          ? 'mdi-check-circle-outline'
+                          : 'mdi-radiobox-blank'
+                      }}
+                    </v-icon>
                   </div>
                 </div>
-                <div v-if="video.filename.endsWith('.webm')" class="flex justify-center w-full text-xs overflow-hidden">
+                <div class="flex flex-row justify-center w-full text-xs overflow-hidden ml-1">
                   <v-tooltip open-delay="500" activator="parent" location="top">{{
-                    video.filename ?? 'Cockpit webm'
+                    video.isProcessed ? 'Processed video' : 'Unprocessed video'
                   }}</v-tooltip>
-                  {{ parseDateFromTitle(video.filename) ?? 'Cockpit webm' }}
+                  {{ parseDateFromTitle(video.fileName) ?? 'Cockpit webm' }}
+                  <v-icon size="10" class="ml-1 mt-[3px]" :class="video.isProcessed ? 'text-green-500' : 'text-red-500'"
+                    >mdi-circle</v-icon
+                  >
                 </div>
               </div>
             </div>
+            <div
+              v-if="availableVideos.length > 1"
+              class="flex flex-row justify-between align-center h-[45px] w-full mb-[-15px]"
+            >
+              <div>
+                <v-btn variant="text" size="small" class="mt-[5px]" @click="toggleSelectionMode">
+                  <v-tooltip open-delay="500" activator="parent" location="bottom"
+                    >Select {{ isMultipleSelectionMode ? 'single' : 'multiple' }} files</v-tooltip
+                  >
+                  {{ isMultipleSelectionMode ? 'Single' : 'Multi' }}</v-btn
+                >
```

Comment:
Not a holder, but I would take a look at how those tags are being formatted. I think you were aiming for something like this:

```html
<v-btn variant="text" size="small" class="mt-[5px]" @click="toggleSelectionMode">
  <v-tooltip open-delay="500" activator="parent" location="bottom">
    Select {{ isMultipleSelectionMode ? 'single' : 'multiple' }} files
  </v-tooltip>
  {{ isMultipleSelectionMode ? 'Single' : 'Multi' }}
</v-btn>
```

This is happening on several parts of this template.
I had this problem before when using the `lint --fix`. It needs to be solved manually (or maybe finding a rule to help the linter).

Reply:
I agree with you. I have manually formatted the code, but looks like lint doesn't like the new format:
![image](https://github.com/bluerobotics/cockpit/assets/14910201/17911557-2c6c-4955-8338-1f476d711a43)

Reply:
Strange. I tried the exactly proposed solution here and the `lint` is not blaming. It's also not blaming on the CI.
Have you checked if the prettier configuration is coming from the repository? We have dedicated rules for it on `package.json`.

Reply:
[CI run](https://github.com/rafaellehmkuhl/cockpit-public/actions/runs/8807605464/job/24175036840#step:5:92) for check.

Reply:
About this one, any luck?